modules/ROOT/pages/install-prereqs.adoc

```diff
@@ -0,0 +1,402 @@
+
+[NOTE]
+====
+Runtime Fabric can reserve up to 0.5 cores per worker node for internal services. This is important 
+when determining the amount of CPU cores and worker nodes needed.
+
+For example, your Runtime Fabric could have three worker nodes and two CPU cores for each worker node. If Runtime Fabric reserves 0.3 cores per worker node, a total of 0.9 cores, the number of available cores displayed in Runtime Manager is 5.1 vCPU instead of 6 vCPU.
+====
+
+Each worker node must have the following resources:
+
+** A minimum of 2 dedicated cores. 
+** A minimum of 15 GiB memory for each dedicated core.
++
+You must consider how many Mules and tokenizers you plan to run on Runtime Fabric, and what they are licensed to deploy. Plan on approximately 0.5 cores per worker node for overhead.
+** A minimum of 80 GiB dedicated disk for the operating system.
+*** A minimum of  20 GiB for `/tmp` directory.
+*** A minimum of 8 GiB for `/opt/anypoint/runtimefabric` directory.
+*** A minimum of 1 GiB for `/var/log/` directory.
+** A minimium of 250 GiB dedicated disk with at least 1000 provisioned IOPS for Docker overlay and other internal services. This translates to a minimum of 4 MBps disk read/write performance.
++ 
+Having 250 GiB ensures there is enough space to run applications, cache docker images, provide temporary storage for running applications, and provide log storage.
+
+=== Additional Resource Requirements for Production Environments
+
+When using Runtime Fabric in a production environment:
+
+* An external load balancer must be configured to load balance external requests to the internal load balancer 
+running on each controller VM, with a health check configured for TCP port 443. A TCP load balancer with a 
+server pool consisting of each of the controller nodes is sufficient.
+* An internal load balancer that runs on at least 3 replicas to maintain availability.
+* Applications serving inbound requests must be scaled to a minimum of 2 replicas.
+
+== Development Configuration Requirements
+
+Runtime Fabric requires a minimum of three nodes in a development environment as shown in the
+following diagram:
+
+image::architecture-development.png[]
+
+[NOTE]
+Runtime Fabric does not support running controller nodes using burstable VMs. When deploying an application on a worker node, you can allocate vCPU for bursting. see xref:deploy-resource-allocation.adoc[Resource Allocation and Performance on Anypoint Runtime Fabric] for additional information.
+
+The following sections describe the minimum system requirements of each controller and worker
+nodes in a development configuration.
+
+=== Controller Nodes
+
+The minimum number of controller nodes required for a non-production environment is one. Even numbers of
+controllers are not supported.
+
+The maximum number of supported controller nodes is 5.
+
+Each controller node must have the following resources:
+
```

Comment:
Ditto previous edits, promoting list items, removing periods, etc.
I will not do these for you here.